.github/workflows/release.yml

```diff
@@ -0,0 +1,124 @@
+name: Release
+on:
+  push:
+    tags:
+      - v*
+
+env:
+  ELIXIR_OPTS: "--warnings-as-errors"
+  ERLC_OPTS: "warnings_as_errors"
+  LANG: C.UTF-8
+
+jobs:
+  create_draft_release:
+    runs-on: ubuntu-18.04
+    env:
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    steps:
+      - name: Create draft release
```

Comment:
I don't know if amount of jobs makes a difference, but since the artifacts are uploaded at the end in a separate `upload_assets` job, the `build_zips` currently doesn't need to depend on `create_draft_release`. Would it make sense to remove `create_draft_release`, and move this step to the existing `upload_assets`? (And maybe rename accordingly).

It would also allow to automatically include the SHA sums in release notes, not only as attached files. (I don't see an easy way to append notes in [gh cli reference](https://cli.github.com/manual/gh_release)

Reply:
> I don't know if amount of jobs makes a difference, but since the artifacts are uploaded at the end in a separate `upload_assets` job, the `build_zips` currently doesn't need to depend on `create_draft_release`. Would it make sense to remove `create_draft_release`, and move this step to the existing `upload_assets`? (And maybe rename accordingly).

Great suggestion! I'll move it to the upload assets step.

> It would also allow to automatically include the SHA sums in release notes, not only as attached files. (I don't see an easy way to append notes in [gh cli reference](https://cli.github.com/manual/gh_release)

It may need to construct a text and put it when creating a release, I thought.